src/manager/RealPrintManager.test.ts

```diff
@@ -133,3 +136,54 @@ test('throws when trying to cancel a job that does not exist', async () => {
     'unable to find job with id=abc123'
   )
 })
+
+test('passes `File` objects through any registered transformers', async () => {
+  const printer = fakePrinter()
+  const manager = new RealPrintManager(printer.name)
+
+  manager.addTransform(
+    // simple transform whose output is a file containing the length of the input as text
+    async (input: File): Promise<File> => ({
+      content: Buffer.from(`${input.content.length}`),
+      contentType: 'text/plain',
+    })
+  )
+
+  await manager.print({
+    content: Buffer.from('a,b,c'),
```

Comment:
Benefit: makes it clear that the values are intended to be literally the same value and prevents having to write them out multiple times. Drawback: introducing abstractions to tests can make them harder to read and understand. I tend to favor local readability in tests over avoiding repetition, i.e. can I understand this test without having to jump all over the file?

When the value is used within a single test there's a reasonable case to be made for extracting to a `const`, though I usually don't just to avoid the temptation of further abstraction.